CHANGELOG.unreleased.md

```diff
@@ -27,3 +27,5 @@
 > - major: if the change breaks compatibility
 >
 > In case of conflict, the highest (lowest in previous list) `$version` wins.
+
+- @xen-orchestra/backups patch
```

Comment:
In theory, this should have been part of the first commit changing `backups`, that's why I commented there, but dont't bother with it for this PR :-)